lte/gateway/python/magma/mobilityd/dhcp_desc.py

```diff
@@ -38,8 +39,10 @@ def __init__(
         self, mac: MacAddress, ip: str,
         state_requested: DHCPState, vlan: int,
         state: DHCPState = DHCPState.UNKNOWN,
-        subnet: str = None, server_ip: str = None,
-        router_ip: str = None, lease_expiration_time: int = 0,
+        subnet: str = None,
+        server_ip: Union[str, IPv4Address, IPv6Address] = None,
+        router_ip: Union[str, IPv4Address, IPv6Address] = None,
+        lease_expiration_time: int = 0,
```

Comment:
The defaults are None so they have to be optional. How about:
```suggestion
        subnet: Optional[str] = None, server_ip: Optional[IPAddress] = None,
        router_ip: Optional[IPAddress] = None, lease_expiration_time: int = 0,
        xid: Optional[str] = None,
```